channels_redis/core.py

```diff
@@ -102,9 +102,9 @@ def push(self, conn):
         """
         Return a connection to the pool.
         """
-        loop = self.in_use[conn]
-        del self.in_use[conn]
+        loop = self.in_use.get(conn)
```

Comment:
```suggestion
        loop = self.in_use.pop(conn, None)
```

Reply:
Indeed it does. Also removed another instance of `del` just to avoid a potential error there too.